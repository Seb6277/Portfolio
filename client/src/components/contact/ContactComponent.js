import React from 'react'
import ContactFormComponent from './ContactFormComponent'

class ContactComponent extends React.Component{
  render() {
    return(
      <div className="contact_frame col-md-8 offset-md-2">
        <h1>Contact</h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam, cumque
          dolorum eaque earum eligendi fuga, id illum omnis placeat praesentium quas
          repudiandae sequi sint unde vel. Corporis earum itaque sit.
        </p>
        <ContactFormComponent/>
      </div>
    )
  }
}

export default ContactComponent