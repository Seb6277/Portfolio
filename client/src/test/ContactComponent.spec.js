import ContactComponent from '../components/contact/ContactComponent'

describe('<ContactComponent/>', () => {
  it('should render corrctly', () => {
    const wrapper = shallow(<ContactComponent/>)
    expect(wrapper).toMatchSnapshot()
  });
})