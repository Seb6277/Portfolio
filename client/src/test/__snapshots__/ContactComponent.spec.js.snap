// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactComponent/> should render corrctly 1`] = `
<div
  className="contact_frame col-md-8 offset-md-2"
>
  <h1>
    Contact
  </h1>
  <p>
    Une question ? Une demande ? Ou juste parlé de la météo ?
    <br />
    C'est ici que ça se passe !
    <br />
    <br />
    Merci de correctement renseigner votre email
  </p>
  <ContactFormComponent />
</div>
`;
